src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -34,9 +36,6 @@ public static void SetOptionsFail()
 
             TestIListNonThrowingOperationsWhenMutable(options.Converters, () => new TestConverter());
 
-            // Verify TypeInfoResolver throws on null resolver
-            Assert.Throws<ArgumentNullException>(() => options.TypeInfoResolver = null);
```

Comment:
make sure to add the attribute to the API review

Reply:
Will do, most likely though we should debate making the getter nullable as well.